appveyor.yml

```diff
@@ -29,7 +29,7 @@ install:
   # protobuf tooling needed at test time. We use a very old version of the compiler
   # because of this: https://github.com/protocolbuffers/protobuf/issues/3957
```

Comment:
Let's remove this notice now that we can get latest

Reply:
Should we get the latest also on the Docker image used for the unix builds on drone.io? Now it's using the v3.8.0.

Reply:
yeah let's consolidate versions across different platforms :+1: